api/users/views.py

```diff
@@ -531,13 +532,15 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, NodesFilterMix
 
     ordering = ('-modified',)
 
+    @newrelic.agent.function_trace()
     # overrides NodesFilterMixin
     def get_default_queryset(self):
         user = self.get_user()
         if user != self.request.user:
             return default_node_list_permission_queryset(user=self.request.user, model_cls=Node).filter(contributor__user__id=user.id)
         return default_node_list_queryset(model_cls=Node).filter(contributor__user__id=user.id)
 
+    @newrelic.agent.function_trace()
```

Comment:
I am not sure what you would see in new-relic after applying this wrapper to these two functions.  Will they clearly be distinguishable from each other? (For example, I wouldn't want to just see `get_queryset` in New Relic, I'd want to see at least something like User Nodes > get_queryset.    Do we need to send any additional info, like a label?


> newrelic.agent.function_trace(name=None, group=None, label=None, params=None, terminal=False)